pkg/bot/slack.go

```diff
@@ -119,11 +123,12 @@ func (b *SlackBot) Start() {
 
 		case *slack.InvalidAuthEvent:
 			log.Error("Invalid Credentials")
-			return
+			return fmt.Errorf("invalid credentials")
```

Comment:
Since we are returning an error now, we should log the error on the caller's side. We should just wrap and return error from here.